mkdocs_alabaster/mkdocs_theme.yml

```diff
@@ -3,5 +3,5 @@
 static_templates:
     - 404.html
 
-include_search_page: false
+include_search_page: true
```

Comment:
That didn't help. Note that the 404 page we're seeing is not the 404.html page shipped with the theme. For some reason, it too is not included.

I've looked at reatthedocs theme, and they seem to have everything set up just the same, but their 404 and search work fine.

BTH I have no clue why that doesn't work :-(

Reply:
Really really really really weird stuff happening here. I'm looking at the [readthedocs](https://github.com/mkdocs/mkdocs/tree/master/mkdocs/themes/readthedocs) theme right now and everything seems to be the same.